restructuredtext_lint/test/test.py

```diff
@@ -23,13 +26,13 @@ def _load_file(self, filepath):
         f.close()
         return file
 
-    def _lint_file(self, *args, **kwargs):
+    def _lint_file(self, *args):
```

Comment:
Instead of removing `**kwargs`, can we fix the invocation to properly pass `**kwargs`?

```
def _lint_file(self, *args, **kwargs):
    return restructuredtext_lint.lint(*args, **kwargs)
```